doc/release/platforms/udp.rst

```diff
@@ -0,0 +1,161 @@
+.. _using-udp:
+
+==============================
+Using the Portable UDP Conduit
```

Comment:
As noted offline, I'm for dropping the extra words in this title (and all other platforms titles) to simply the name of the platform, e.g. "UDP Conduit"